doc/reference.rst

```diff
@@ -42,3 +53,215 @@ It then runs the resulting buildout.
+  that the configuration containing the ``extends`` option should
+  :ref:`extend <extends_option>`. The names may be file paths, or
+  URLs.  If they are relative paths, they are interpreted relative to
+  the configuration containing the ``extends`` option.
+
+.. _extends-cache-buildout-option:
+
+
+extends-cache
+  An optional directory to cache remote configurations in.  This
+  allows remote configurations to be used in :ref:`offline mode
+  <offline-mode>`.  When not in offline mode, and a remote
+  configuration is used, by using a URL in an
+  :ref:`extends option <extends_option>`, the configuration is written
+  to the extends cache and will be used if the buildout is later run
+  in offline mode.
+
+  If the value is a relative path and doesn't contain value
+  substitutions, it's interpreted relative to the directory containing
+  the configuration file that defined the value. (If it contains value
+  substitutions, and the result is a relative path, then it will be
+  interpreted relative to the buildout directory.)
+
+find-links, default: ''
+  Extra locations to search for distributions to download.
+
+  These may be file paths or URLs.  These may name individual
+  distributions or directories containing
+  distributions. Subdirectories aren't searched.
+
+index
+  An alternate index location.
+
+  This can be a local directory name or an URL.  It can be a flat
+  collection of distributions, but should be a "simple" index, with
+  subdirectories for distribution `project names
+  <https://packaging.python.org/distributing/#name>`_ containing
+  distributions for those projects.
+
+  If this isn't set, then ``https://pypi.python.org/simple/`` is used.
+
+install-from-cache, default: 'false'
+  If this is true, then distributions will only be installed if they
+  can be found in the :ref:`download cache <download-cache>`.
+
+installed, default: '.installed.cfg'
+  The name of the file used to store information about what's installed.
+
+  Buildout keeps information about what's been installed so it can
+  remove files created by parts that are removed and so it knows
+  whether to update or install new parts from scratch.
+
+  If this is a relative path, then it's interpreted relative to the
+  buildout directory.
+
+log-format, default: ''
+  Format to use for log messages.
+
+log-level, default: 'INFO'
+  The default level to log at.
+
+newest, default: 'true'
+  If true, check for newer distributions.  If false, then only look
+  for distributions when installed distributions don't satisfy requirements.
+
+.. _offline-mode:
+
+offline, default: 'false'
+  If true, then offline mode is enabled.
+
+  In offline mode, no network requests should be made.  It's the
+  responsibility of recipes to adhere to this.  Recipes that would
+  need to download files may use the :ref:`download
+  cache <download-cache>`.
+
+  No distributions are installed in offline mode. If installed
```

Comment:
> No distributions are installed in offline mode.

Even if they're present in the download cache?  Even if `install-from-cache` is set to `true`?

Clearly I don't understand what `offline` and `install-from-cache` actually do :/